docs/tutorials/data-loading/local-files.md

```diff
@@ -0,0 +1,131 @@
+    into Qlik Assocative Engine without having to write load scripts. The tutorial however, focuses on direct use of
+    load scripts in order to explain and show their potential.
+
+## Prerequisites
+
+To follow along in this tutorial, you should have basic understanding of Docker.
+
+You need the following software installed:
+
+* Git
+* Docker
+* docker-compose
+* Node.js
+
+## Setup
+
+To run the example code, clone the [qlik-oss/core-data-loading](https://github.com/qlik-oss/core-data-loading) Git
+repository. Check out the repo documentation to get familiar with the content and structure.
+
+Only one instance of Qlik Associative Engine is needed and it is started with all necessary data files mounted.
+Change `ACCEPT_EULA` as applicable and run:
+
+```sh
+ACCEPT_EULA=<yes/no> docker-compose up -d
+```
+
+Now install dependencies:
+
+```sh
+npm install
+```
+
+## Sample data files
+
+The examples use these files to load data from:
+
+* [airports.csv](https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.csv) -
+  Headers on first row, comma-separated data
+* [airports.xlsx](https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.xlsx) -
+  Headers on first row, MS Excel Open XML
+
+The files are mounted into the Qlik Associative Engine and made available on the file system visible from the container.
+
+## Examples
+
+### CSV file
+
+This example uses the [load-csv-file](https://github.com/qlik-oss/core-data-loading/blob/master/scripts/load-csv-file)
+load script to load data from the
+[airports.csv](https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.csv) file into Qlik Associative
+Engine.
+
+Script characteristics:
+
+* Names the resulting table `Airports`
+* Loads data from all headers (`*` notation)
+* Specifies that the file is in text format and `utf8` encoded
+* Sets the value delimiter to the `,` character
+* Uses embedded lables from the CSV file itself as field names, reading labels from the first line by default
+
+```sh
+npm start load-csv-file
+```
+
+The exepected output is a list of all airports loaded with field names as given by the headers in the CSV file. Also
+study the [code](https://github.com/qlik-oss/core-data-loading/blob/master/index.js) for invoking the load script using
+[enigma.js](https://github.com/qlik-oss/enigma.js/).
+
+### Excel file
+
+This example is similar to the previous example, but now uses the
+[load-xlsx-file](https://github.com/qlik-oss/core-data-loading/blob/master/scripts/load-xlsx-file) load script to load
+the same airport data from the
+[airports.xlsx]((https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.xlsx)) Excel file instead.
+
+Script characteristics:
+
+* Names the resulting table `Airports`
+* Loads data from all headers (`*` notation)
+* Specifies that the file is in in Excel format (`ooxml`)
+* Uses embedded lables from the Excel file itself as field names
```

Comment:
labels

Reply:
Same here. In QAE it results in field names.

Reply:
Sorry, it says 'lables' and should be labels